Tests/test_TCoffee_tool.py

```diff
@@ -93,25 +100,37 @@ def test_TCoffee_clustalw(self):
         cmdline.outorder = "input"
         cmdline.set_parameter("gapext", -5)
         cmdline.type = "protein"
-        self.assertEqual(str(cmdline), t_coffee_exe + " -output clustalw_aln "
-                         "-infile Fasta/fa01 -outfile Fasta/tc_out.aln "
-                         "-type protein -outorder input -gapopen -2 -gapext -5")
+        self.assertEqual(
+            str(cmdline),
+            t_coffee_exe + " -output clustalw_aln "
+            "-infile Fasta/fa01 -outfile Fasta/tc_out.aln "
+            "-type protein -outorder input -gapopen -2 -gapext -5",
+        )
```

Comment:
Can probably save a line by changing where the long string is split?

Reply:
black splits on the addition, it didn't save a line but looks more standard.